src/components/molecules/MultiSelect.tsx

```
@@ -0,0 +1,220 @@
+            );
+        else if (
+            selected.length > 0 &&
+            selected[selected.length - 1].value !== allOption.value &&
+            JSON.stringify(selected.sort(comparator)) !==
+            JSON.stringify(filteredOptions)
+        )
+            return props.onChange(selected);
+        else
+            return props.onChange([
+                ...props.value?.filter(
+                    ({ label }: Option) =>
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)